dashboard/src/components/AlertListItem.tsx

```
@@ -54,7 +54,7 @@ const AlertListItem: React.FC<AlertListItemProps> = ({ session, onClick, searchT
           variant="body2" 
           sx={{ fontWeight: 500 }}
           dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)